planning/behavior_path_start_planner_module/src/start_planner_module.cpp

```diff
@@ -384,17 +384,18 @@ BehaviorModuleOutput StartPlannerModule::plan()
           getLogger(), *clock_, 5000, "Insert stop point in the path because of dynamic objects");
         path = *stop_path;
         status_.prev_stop_path_after_approval = std::make_shared<PathWithLaneId>(path);
-        status_.has_stop_point = true;
+        status_.stop_pose = stop_pose_;
       } else {
         path = current_path;
       }
-    } else if (!isWaitingApproval() && status_.has_stop_point) {
+    } else if (!isWaitingApproval() && status_.stop_pose) {
       // Delete stop point if conditions are met
       if (status_.is_safe_dynamic_objects && isStopped()) {
-        status_.has_stop_point = false;
+        status_.stop_pose = std::nullopt;
         path = getCurrentPath();
       }
       path = *status_.prev_stop_path_after_approval;
+      stop_pose_ = status_.stop_pose;
```

Comment:
stop_pose_ is shared_ptr and can be nullptr, so using only stop_pose_ directly also good for me.
```
std::shared_ptr<geometry_msgs::msg::Pose> stop_pose_ptr_;
```

Reply:
@kosuke55 
>using only stop_pose_ directly also good for me

You mean without defining stop_pose in status?

If so, in this cycle, stop_pose is not calculated, so necessary to assign previously calculated stop_pose to `stop_pose_`.
checking if stop_pose is already inserted in this [statement](https://github.com/autowarefoundation/autoware.universe/pull/5974/files/bc055b0415c573412fc2559453aaa4c4e9388074#diff-6f85c7d1beff552f9efeab67102f536fd1222a2903286b3dcc8c5413a4b8e7f3R391)

Reply:
yes, if it is necessary it's ok! thanks!